dags/resources/stages/parse/table_definitions/abracadabra/BentoBoxMarkets_event_LogAccrue.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": false,
+                    "internalType": "uint128",
+                    "name": "accruedAmount",
+                    "type": "uint128"
+                }
+            ],
+            "name": "LogAccrue",
+            "type": "event"
+        },
+        "contract_address": "SELECT cloneAddress FROM ref('BentoBoxV1_event_LogDeploy')",
```

Comment:
This should be in a second PR.

 The first PR will have all the base tables, ie: BentoBoxV1_event_LogDeploy , which we can deploy first so these won't fail because the table doesn't exist yet.